libkbfs/tlf_journal.go

```diff
@@ -1693,9 +1698,12 @@ func (j *tlfJournal) putBlockData(
 	case nil:
 		// Continue.
 	case context.DeadlineExceeded:
+		usageBytes, limitBytes, usageFiles, limitFiles :=
+			j.diskLimiter.getDiskLimitInfo()
```

Comment:
There's a tiny race here, and it's possible that this call won't show usage being over the limit if a flush completed between the acquire and now, but I don't think it's worth changing the signature of `beforeBlockPut` for.  But what do you think?